src/tag/constants.js

```diff
@@ -5,14 +5,12 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
-/* eslint-disable import/prefer-default-export */
-import {LightTheme} from '../themes';
 
-export const STYLE = {
-  primary: LightTheme.colors.primary400,
-  warning: LightTheme.colors.warning400,
-  positive: LightTheme.colors.positive400,
-  negative: LightTheme.colors.negative400,
+export const COLOR_STYLE_KEYS = {
```

Comment:
Not sure if we still need this constant, but we should add/export the `KIND` constant as well.
```
export const KIND = {
  primary: 'primary',
  positive: 'positive',
  warning: 'warning',
  negative: 'negative',
};
```